tests/test_input_module.py

```diff
@@ -36,6 +39,8 @@
 TEST_DIR = Path(__file__).parent
 TMP_DIR = TEST_DIR / "tmp"
 TMP_DIR.mkdir(exist_ok=True)
+DAT_DIR = TEST_DIR / "data" / "test-inputs"
+DAT_DIR.mkdir(exist_ok=True)
```

Comment:
Minor thing, but can we name this TEST_DATA_DIR? I'm hesitant to use DAT here just since that has such a niche QUIC-Fire context.

Reply:
Sure thing